script.lua

```diff
@@ -6507,6 +6505,20 @@ function setupSwapButtons(obj, params)
         },
         children = {},
     })
+    table.insert(xml, {
+        tag = "VerticalLayout",
+        attributes = {
+            id = "GainSpirits",
+            recurse = "GainSpirits",
```

Comment:
this is a top level xml element right? you shouldn't need the recurse tag here

Reply:
It does, since it only checks the id after checking recuse attribute.